frontend/components/bc/toggle/MultiBarButton.vue

```diff
@@ -17,16 +19,33 @@ const icon = computed(() => {
 </script>
 
 <template>
-  <ToggleButton v-model="selected" class="bc-toggle" on-label="''" off-icon="''">
-    <template #icon="slotProps">
-      <slot name="icon" v-bind="slotProps">
-        <FontAwesomeIcon v-if="icon" :icon="icon" />
-      </slot>
+  <BcTooltip>
+    <template #tooltip>
+      <div class="button-tooltip">
+        <div>{{ selected ? $t('filter.enabled'): $t('filter.disabled') }}</div>
+        <div v-if="tooltip" class="individual">
+          {{ tooltip }}
+        </div>
+      </div>
     </template>
```

Comment:
**Bug:** Clicking the button now forces the tooltip to be visible until the users clicks somewhere else again.

Reply:
fixed